crates/rpc-types/src/eth/transaction/request.rs

```diff
@@ -36,8 +39,11 @@ pub struct TransactionRequest {
     /// EIP-2718 type
     #[serde(rename = "type")]
     pub transaction_type: Option<U8>,
-    /// sidecar for EIP-4844 transactions
-    pub sidecar: Option<BlobTransactionSidecar>,
```

Comment:
we still need this if we want to use this to convert into a full eip4844 transaction.

it's not needed for anything rpc besides eth_sendTransaction, but if we wan to use this type as the builder for constructing sendable transactions we need this

Reply:
thanks, that's what i was asking about in alloy maintainers, will add it back.